client/src/app/admin/panel/staff/admin-panel-departments.js

```diff
@@ -144,21 +259,37 @@ class AdminPanelDepartments extends React.Component {
         }
     }
 
+    onDefaultDepartmentFormSubmit(defaultDepartmentFormValue) {
+        let publicDepartments = _.filter(SessionStore.getDepartments(), function(item) { return item.private != 1} ); 
+
+        this.setState({formLoading: true, edited: false});
```

Comment:
`formLoading` and `editing` were being used for the other form, it's conflicting with it.
I suggest to divide this component in two components, one for department edting and other for default defaultment editing